content/it/zero-trust-architecture.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Architettura Zero Trust
+status: Completed
+category: Concetto
+---
+
+## Cos'è
+L'Architettura Zero Trust prescrive un approccio alla progettazione e all'implementazione dei sistemi IT in cui la fiducia viene completamente rimossa. Il principio fondamentale è "non fidarti mai, verifica sempre", i dispositivi o i sistemi stessi, mentre comunicano con altri componenti di un sistema, si verificano sempre prima di farlo. In molte reti oggi, all'interno della rete aziendale, i sistemi e i dispositivi all'interno possono comunicare liberamente tra loro poiché si trovano all'interno del confine attendibile del perimetro della rete aziendale. L'architettura Zero Trust adotta l'approccio opposto in cui, sebbene all'interno del perimetro della rete, i componenti all'interno del sistema devono prima superare la verifica prima che venga effettuata qualsiasi comunicazione.
```

Comment:
```suggestion
L'Architettura Zero Trust prescrive un approccio alla progettazione e all'implementazione dei sistemi IT in cui la fiducia viene completamente rimossa. Il principio fondamentale è "non fidarti mai, verifica sempre": i dispositivi o i sistemi stessi, mentre comunicano con altri componenti di un sistema, si verificano sempre prima di farlo. In molte reti oggi, all'interno della rete aziendale, i sistemi e i dispositivi all'interno possono comunicare liberamente tra loro poiché si trovano all'interno del confine attendibile del perimetro della rete aziendale. L'architettura Zero Trust adotta l'approccio opposto in cui, sebbene all'interno del perimetro della rete, i componenti all'interno del sistema devono sottoporsi a una verifica prima di effettuare qualsiasi comunicazione.
```

Reply:
- `i dispositivi o i sistemi stessi, mentre comunicano con altri componenti di un sistema, si verificano sempre prima di farlo.` ho letto prima l'italiano non capendo chi verificasse chi, al confronto con la sorgente si è chiarito il concetto.

a mio parere dovremmo esplicitare il riflessivo: "i dispositivi o i sistemi stessi, mentre comunicano con altri componenti di un sistema, provvedono sempre ad un'auto verifica prima di procedere con la comunicazione"

- _In molte reti oggi, all'interno della rete aziendale, i sistemi e i dispositivi all'interno possono comunicare liberamente tra loro poiché si trovano all'interno del confine attendibile del perimetro della rete aziendale._

Oggi, in molte reti all'interno della rete aziendale, i sistemi e i dispositivi contenuti possono comunicare liberamente tra loro poiché si trovano dentro il perimetro del confine attendibile del perimetro della rete aziendale._

Reply:
@sistella ho fatto un errore! è "dentro il confine attendibile del perimetro della rete" (ho usato due volte perimetro inutilmente, scusa)

Reply:
updated! @annalisag-spark